stackgl_modules/package.json

```diff
@@ -13,21 +13,22 @@
     "box-intersect": "plotly/box-intersect#v1.1.0",
     "convex-hull": "^1.0.3",
     "delaunay-triangulate": "^1.1.6",
-    "gl-axes3d": "^1.7.0",
-    "gl-cone3d": "^1.6.0",
-    "gl-error3d": "^1.0.16",
+    "gl-axes3d": "github:hborchardt/gl-axes3d#83caafc058b2d60b6006e26432967c411c125fe1",
+    "gl-cone3d": "github:hborchardt/gl-cone3d#e9728cd068e0dfe11b92c5785974cd463f78c01c",
```

Comment:
Please open pull requests on all `gl-vis/*` dependencies  and paste the links for every line change in `package.json` file. Thank you!

Reply:
See https://github.com/gl-vis/gl-cone3d/pull/21

Reply:
Now: `gl-cone3d@1.6.1`